autogen.sh

```diff
@@ -273,6 +283,14 @@ for arg in "$@" ; do
 	    autotoolsdir=`echo "A$arg" | sed -e 's/.*=//'`
 	    ;;
 
+    -without-ofi|--without-ofi|-without-libfabric|--without-libfabric)
+        do_ofi=no
+        ;;
+
+    -without-ucx|--without-ucx)
+        do_ucx=no
+        ;;
+
```

Comment:
Shouldn't `do_ofi` and `do_ucx` be set *before* the above checks?

Reply:
Only if you want it to work... 😄

Reply:
Fixed.